### PR TITLE
fix(shell): correct /usage remaining quota colors

### LINKED ISSUE #2019: Wrong usage color
### What version of Kimi Code CLI is running?

1.37.0

### What issue are you seeing?

It uses wrong color in the usage command.

The color is chosen by the remaining ratio:

https://github.com/MoonshotAI/kimi-cli/blob/34025f6af57c60de5b039673dfd607f18e2e903b/src/kimi_cli/ui/shell/usage.py#L255-L257
https://github.com/MoonshotAI/kimi-cli/blob/34025f6af57c60de5b039673dfd607f18e2e903b/src/kimi_cli/ui/shell/usage.py#L276-L281

### What is the expected behavior?

```diff
+ # remaining ratio is passed in
  def _ratio_color(ratio: float) -> str:
-     if ratio >= 0.9:
+     if ratio <= 0.1:
          return "red"
-     if ratio >= 0.7:
+     if ratio <= 0.3:
        return "yellow"
     return "green"
```

### PR DESCRIPTION
**Pull request by binichallein**
## Related Issue

Resolve #2019

## Description

This fixes the inverted quota colors in `/usage`.

Root cause:

* `_ratio_color()` receives the remaining-quota ratio (`(limit - used) / limit`), not the used ratio
* the thresholds were previously mapped as if larger values were more dangerous, so high remaining quota rendered red and low remaining quota rendered green

Changes:

* map `<= 10% left` to `red`
* map `<= 30% left` to `yellow`
* map `> 30% left` to `green`
* add a regression test covering remaining-quota thresholds and over-limit behavior

This is equivalent to the common used-quota thresholds:

* `< 70% used` => green
* `70%-89% used` => yellow
* `>= 90% used` => red

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/MoonshotAI/kimi-cli/blob/main/CONTRIBUTING.md) document.
- [x] I have linked the related issue, if any.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] I have run `make gen-changelog` to update the changelog.
- [x] I have run `make gen-docs` to update the user documentation.

---

<a href="https://app.devin.ai/review/moonshotai/kimi-cli/pull/2039" target="_blank">
  <picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://static.devin.ai/assets/gh-open-in-devin-review-dark.svg?v=1">
    Open in Devin Review
  </picture>
</a>

### CODE REVIEW

File: docs/en/release-notes/changelog.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ This page documents the changes in each Kimi Code CLI release.
 
 ## Unreleased
 
+- Shell: Fix usage remaining quota colors
+
```
[the remark this answers is not in the document]
Comment (binichallein) [reply docs/en/release-notes/changelog.md:25]: I initially resolved the changelog conflict manually, but I corrected the workflow afterward.

  The current state follows the repo rules:
  - `CHANGELOG.md` is the source of truth
  - `docs/en/release-notes/changelog.md` was regenerated via `node docs/scripts/sync-changelog.mjs`
  - `docs/zh/release-notes/changelog.md` was merged accordingly as the translated page

**Comment by sunmy2019** [comment]
You misunderstood the problem and gave a wrong fix.

**Comment by sunmy2019** [comment]
Usually people gave

0 < used < 70%, green
70% < used < 90%, yellow
90% < used < 100%, red

This is common practice and claude code also obeys.

> green under 70%, yellow 70-89%, red 90%+

https://code.claude.com/docs/en/statusline#display-multiple-lines

**Comment by binichallein** [comment]
@sunmy2019 Thanks for catching this. You were right that my initial fix misunderstood the metric.

I've updated the PR so `_ratio_color()` now uses the remaining quota ratio correctly:
- `<= 10% left` => `red`
- `<= 30% left` => `yellow`
- `> 30% left` => `green`

That matches the usual used-quota thresholds after converting from remaining quota. I also added regression tests and resolved the changelog/docs conflicts following the repo workflow (`CHANGELOG.md` -> sync English changelog -> update Chinese changelog).

Latest update is in `4137323a`. Please take another look when convenient.

**Comment by wbxl2000** [comment]
Thanks for iterating on this after the reviewer feedback, and for adding regression coverage. The updated direction here matches the behavior we want: low remaining quota should be yellow/red, high remaining quota should stay green.

We have now merged #2078, which covers this fix plus the related remaining-quota progress bar fill, full-quota `finished_style`, zero/negative/over-used quota edge cases, and broader tests. #2019 is closed by that merge, and #2078 credits this PR among the related fixes that helped converge on the final behavior.

I am closing this as superseded by #2078 so we do not leave a stale duplicate PR open. Appreciated.

**Comment by wbxl2000** [comment]
One more maintainer note: I really appreciate the follow-up work here, especially the correction after review feedback and the regression test. We ended up merging #2078 because it bundled this fix with a few adjacent rendering edge cases and broader coverage, but this PR absolutely helped converge on the final behavior.

Please do not read the close as a rejection of the work. This was a duplicate-resolution call after the fix had landed, not a judgment that this PR was not valuable. Thanks again for helping move this issue forward.
